project-base/storefront/hooks/comparison/useComparison.tsx

```diff
@@ -9,58 +8,43 @@ import {
 import { getUserFriendlyErrors } from 'helpers/errors/friendlyErrorMessageParser';
 import { useTypedTranslationFunction } from 'hooks/typescript/useTypedTranslationFunction';
 import { useCurrentUserData } from 'hooks/user/useCurrentUserData';
-import { Dispatch, SetStateAction, useEffect, useMemo, useState } from 'react';
+import { useEffect, useState } from 'react';
 import { usePersistStore } from 'store/zustand/usePersistStore';
 
-export const useHandleCompare = (
-    productUuid: string,
-): {
-    isProductInComparison: boolean;
-    handleProductInComparison: () => void;
-    handleRemoveAllFromComparison: () => void;
-    isPopupCompareOpen: boolean;
-    setIsPopupCompareOpen: Dispatch<SetStateAction<boolean>>;
-    getComparisonProducts: () => ComparedProductFragmentApi[];
-} => {
+export const useComparison = () => {
     const t = useTypedTranslationFunction();
-    const [isPopupCompareOpen, setIsPopupCompareOpen] = useState(false);
     const { isUserLoggedIn } = useCurrentUserData();
-    const [isProductInComparison, setIsProductInComparison] = useState(false);
     const [, addProductToComparison] = useAddProductToComparisonMutationApi();
     const [, removeProductFromComparison] = useRemoveProductFromComparisonMutationApi();
     const [, cleanComparison] = useCleanComparisonMutationApi();
-    const productsComparisonUuid = usePersistStore((store) => store.productsComparisonUuid);
+    const comparisonUuid = usePersistStore((store) => store.productsComparisonUuid);
```

Comment:
Previous approach with the names matching seemed better to me. Why did we rename it?

Reply:
Because we don't use "productComparison" in none of the components neither functions. So this was about unification. But I've added one more change which I haven't noticed - rename `productsComparisonUuid` value stored in Persist store to "comparisonUuid" to match the rest of the application.

Reply:
I do not see this changed